devops/aws/serverless/main.tf

```diff
@@ -507,6 +524,14 @@ module "create_user_lambda" {
   }
 
   attach_dead_letter_policy = false
+  attach_policy_statements = true
+  policy_statements = {
+    sqs = {
+      effect    = "Allow",
+      actions   = ["sqs:*"],
+      resources = ["arn:aws:sqs:${var.aws_region}:${var.aws_account_id}:${aws_sqs_queue.users_demo_post_queue.name}"]
```

Comment:
Maybe we should stop naming our services/ queues etc using demo or tutorial just with normals names as if they were normal services. What do you think ?